adoc/bp_chap_expanded_support.adoc

```diff
@@ -383,3 +383,94 @@ See the matrix of {productname} clients on the main page of the {productname} wi
+However, {ubuntu} lacks a root user by default, so in order to use bootstrapping, you will require an existing user with [command]``sudo`` privileges for Python.
+
+Other supported features:
+
+* Synchronizing [systemitem]``.deb`` channels
+* Assigning [systemitem]``.deb`` channels to minions
+* GPG signing [systemitem]``.deb`` repositories
+* Minion cleanup on delete
+* Information displayed in System details pages
+* Package install, update, and remove
+* Package install using [command]``Package States``
+* Configuration & state channels
+
+Some actions are not yet supported, however these may become available in a later version:
+
+* Patch and errata support
+* Bare metal installations, PXE booting, and virtual host provisioning
+* Live patching
+
+Note that {ubuntu} clients must be Salt minions.
+Traditional clients running {ubuntu} are not supported.
+
+This section describes how to manually prepare {ubuntu} clients for registration to your {susemgr} Server and synchronize the repositories.
+
+
+[[bp.ubuntu.register]]
+== Registering {ubuntu} Clients
+
+
+sudo vi /etc/apt/sources.list.d/suma_client_tools.list
+
+# add next line for Ubuntu 16.04
+deb http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/3.2:/Ubuntu16.04-SUSE-Manager-Tools/xUbuntu_16.04/ /
+
+# add next line for Ubuntu 18.04
+deb http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/3.2:/Ubuntu18.04-SUSE-Manager-Tools/xUbuntu_18.04/ /
+
+# Ubuntu 16.04: import key
+curl http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/3.2:/Ubuntu16.04-SUSE-Manager-Tools/xUbuntu_16.04/Release.key | sudo apt-key add -
+
+# Ubuntu 18.04: import key
+curl http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/3.2:/Ubuntu18.04-SUSE-Manager-Tools/xUbuntu_18.04/Release.key | sudo apt-key add -
+
+sudo apt update
+
+sudo visudo
+# add next line, where <user> will be used to bootstrap the minion from the UI
+<user>   ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3
+
+
+
+[[bp.ubuntu.synchrepo]]
+== Synchronizing {ubuntu} Repositories
+
+
+Software -> Manage Software Channels -> Manage Repositories -> Create Repository
+Repository Type: deb
+Repository URL: url to directory containing binarires, e.g for source
+   http://archive.ubuntu.com/ubuntu bionic restricted main => http://archive.ubuntu.com/ubuntu/dists/bionic/main/binary-amd64/
+   http://archive.ubuntu.com/ubuntu bionic-updates restricted main =>  http://us.archive.ubuntu.com/ubuntu/dists/bionic-updates/main/binary-amd64
+
+Software -> Manage Software Channels -> Create Channel
+Architecture: AMD64 Debian 
+... 
+Click "Create"
+
+Software -> Manage Software Channels -> click new channel -> Repositories tab -> Add/Remove tab
+Select repo
+Click "Update Repositories"
+Select Sync tab
+Click "Sync Now"
+
+Check sync progess: tail -f /var/log/rhn/reposync.log /var/log/rhn/reposync/*
+
+Unsupported Helper script
```

Comment:
The helper script if for internal use/testing only. I'd suggest to remove it from the docs.